controls/cis_rhel8.yml

```diff
@@ -1215,6 +1215,15 @@ controls:
       - sysctl_net_ipv4_conf_default_log_martians
       - sysctl_net_ipv4_conf_default_log_martians_value=enabled
 
+  - id: 3.3.10
+    title: Ensure TCP SYN Cookies is enabled (Automated)
```

Comment:
The policy uses lower cases in this title. I also think it would be better as you proposed, but maybe we should keep it aligned to the policy title.

Reply:
I was considering that as well. I will update to match the policy.